### ISSUE TITLE
Schools not keyboard accessible

### ISSUE THREAD

**Issue by kburk1997** [YACS-RCOS/yacs#301]
I am testing YACS with ChromeVox, and I was able to navigate to schools with a keyboard, but could not click the link with only a keyboard. This along with #299 makes YACS unusable for screen reader users and sighted keyboard/sip-and-puff users.

**Comment by Chromelia** [comment]
This should be fixed wtih #480.
